sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/exemplar/ExemplarReservoirFactory.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.exemplar;
+
+import io.opentelemetry.sdk.metrics.view.Aggregation;
+
+/** A factory for {@link ExemplarReservoir}s that can leverage Aggregation configuration. */
+public interface ExemplarReservoirFactory {
+  /**
+   * Constructs a new sampling reservoir for a given metric stream.
```

Comment:
I think this comment needs to be fleshed out a bit. Something like :
```
Constructs a new sampling reservoir. This will be applied to a single metric stream and will be responsible
for capturing Exemplar data associated with the metric recordings.
```

Reply:
Added your description exactly for now.  If you have any better descriptive ideas, let me know.